db/db_impl_compaction_flush.cc

```diff
@@ -2710,7 +2713,7 @@ Status DBImpl::BackgroundCompaction(bool* made_progress,
 
   if (status.ok() || status.IsCompactionTooLarge()) {
     // Done
-  } else if (status.IsShutdownInProgress()) {
+  } else if (status.IsShutdownInProgress() || status.IsColumnFamilyDropped()) {
```

Comment:
Do not remember if I have made this comment.
Can status be `ShutdownInProgress`, since it's the return value of `CompactionJob::Install`.